pages/add/index.js

```diff
@@ -25,15 +26,20 @@ export default function AddEntries({
   handleAddEntry,
   wordList,
   handleActivePage,
+  activeUser,
+  handleActiveUser,
 }) {
+  const { data: session } = useSession();
+
   useEffect(() => {
     handleActivePage("add");
-  }, [handleActivePage]);
+    handleActiveUser(session?.user._id);
+  }, [handleActivePage, handleActiveUser, session]);
```

Comment:
is there a reason why you perform this setting of the Active user here and not in the _app.js? And when you use the useRouter hook you can get the fullPath which might solve your need for the `handleActivePage`.

Reply:
Now that I think of it, no good one. I just tried now and can't really make it work. I get an error message when I use "useSession" because it needs to be inside of a <SessionProvider> it says. 
I already removed the activePage though.

Reply:
I was able to move it to Layout and just call it once. I'm not sure if that is best practice. I couldn't make it work in _app.js